### ISSUE TITLE
Refactor with CanvasEditor

### ISSUE THREAD

**Issue by tjhickey724** [tjhickey724/collabed#15]
Rewrite the code so that it uses Javascript classes.. So we can create an Editor class associated to a canvas and have several editors on a single webpage.

**Comment by tjhickey724** [comment]
We have refactored it so that there is a CanvasEditor class.
Next we will create a class which maintains the window on the text document
(rows and cols visible, row and col offset, total text object) This is another story...

Once that is done we can abstract the text object using MSET
and then we can add the listeners from the collabed package....
